backend/hostgw/network.go

```diff
@@ -45,6 +45,10 @@ func (n *network) MTU() int {
 	return n.extIface.Iface.MTU
 }
 
+func (n *network) SubnetFileVars() string {
```

Comment:
It's cleaner to return `map[string]string` and render it to text in the function that writes out the file.